main.go

```diff
@@ -108,7 +108,7 @@ type (
 		BillConnectID     string  `env:"BILL_CONNECT_ID"`
 		Shard             string  `env:"SHARD" envDefault:"NAM"`
 		KubecostHost      string  `env:"KUBECOST_HOST" envDefault:"localhost:9090"`
-		Aggregation       string  `env:"AGGREGATION" envDefault:"namespace"`
+		Aggregation       string  `env:"AGGREGATION" envDefault:"pod"`
```

Comment:
I understand that we are adding support for au..

What is this change  envDefault:"pod"`?

Reply:
Changed default aggregate value to "pod" (if environment variable is not present)